packages/sdk/src/credential/teamsUserCredential.browser.ts

```diff
@@ -280,10 +265,6 @@ export class TeamsUserCredential implements TokenCredential {
         clientId: this.config.clientId!,
         authority: `https://login.microsoftonline.com/${this.tid}`,
       },
-      cache: {
-        cacheLocation: "sessionStorage",
```

Comment:
since we states that we caches token in session storage, can we keep the cacheLocation config in case MSAL changes the default config in the future?

Reply:
Fixed. Thx!